### PR TITLE
Rename Service APIs to Gateway API

### PR DESCRIPTION
**Pull request by shamsher31**
To fix https://github.com/istio/istio.io/issues/9338

Ref: https://github.com/kubernetes-sigs/gateway-api#kubernetes-gateway-api

### CODE REVIEW

**Review by craigbox — APPROVED** [review]

File: content/en/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 1]
```diff
@@ -1,25 +1,27 @@
 ---
-title: Kubernetes Service APIs
-description: Describes how to configure the Kubernetes Service APIs with Istio.
+title: Kubernetes Gateway API
+description: Describes how to configure the Kubernetes Gateway API with Istio.
 weight: 50
+aliases:
+    - /docs/tasks/traffic-management/ingress/service-api/
```
Comment (craigbox) [inline content/en/docs/tasks/traffic-management/ingress/gateway-api/index.md:6]:
```suggestion
    - /docs/tasks/traffic-management/ingress/service-apis/
```

**Review by craigbox — CHANGES_REQUESTED** [review]

**Review by craigbox — APPROVED** [review]

**Comment by craigbox** [comment]
Lint is failing on /docs/tasks/traffic-management/ingress/service-apis/ being 404, but there's an alias added for this path.  Thoughts on how to fix, @ericvn @frankbu @davidhauck ?

**Comment by ericvn** [comment]
My first thought is formatting of the alias. Try updating it locally and running `lint` to verify.

**Comment by shamsher31** [comment]
> My first thought is formatting of the alias. Try updating it locally and running `lint` to verify.

I think the alias only works for browser redirect and we will have to fix the 404 URL manually (I did this previously) and it will work.

**Pull request merged**